homeassistant/components/anomaly/binary_sensor.py

```diff
@@ -82,20 +60,21 @@
 )
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {vol.Required(CONF_SENSORS): cv.schema_with_slug_keys(SENSOR_SCHEMA)}
+    {vol.Required(const.CONF_SENSORS): cv.schema_with_slug_keys(SENSOR_SCHEMA)}
 )
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
-    """Set up the anomaly sensors."""
-    await async_setup_reload_service(hass, DOMAIN, PLATFORMS)
+async def _async_create_entities(hass: HomeAssistant, config: OrderedDict):
+    """Create the template binary sensors."""
 
+    config_sensors = config.get(const.CONF_SENSORS, {})
```

Comment:
This is a required item.
```suggestion
    config_sensors = config[const.CONF_SENSORS]
```